### ISSUE TITLE
cant run with conda on mac

### ISSUE THREAD

**Issue by sugizo** [microsoft/playwright-python#517]
```
$ uname
Darwin
$ conda -V
conda 4.9.2
$ pip list | grep playwright
playwright                    0.162.2
$ playwright codegen
-bash: playwright: command not found
$ python
Python 3.8.5 (default, Sep  4 2020, 02:22:02) 
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from playwright.sync_api import sync_playwright
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'sync_playwright' from 'playwright.sync_api' (/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/site-packages/playwright/sync_api.py)
```

**question**
any idea how to use playwright with conda on mac ?

thanks

**Comment by kumaraditya303** [comment]
Currently `conda` is not available for playwright, See https://github.com/microsoft/playwright-python/issues/257

**Comment by holg** [comment]
yes there is no conda package, but it works well via pip:
pip install playwright==1.8.0a1
I am using conda @python3.8.5 without problems,
but i am using playwright 1.8.0a1:
uname
Darwin
conda -V
conda 4.9.2
pip list |grep  playwright                                  
playwright           1.8.0a1
 playwright codegen
from playwright.sync_api import sync_playwright

def run(playwright):
    browser = playwright.chromium.launch(headless=False)
    context = browser.new_context()

    # Open new page
    page = context.new_page()

    # Close page
    page.close()

    # ---------------------
    context.close()
    browser.close()

**Comment by kumaraditya303** [comment]
Yes, It works with pip because pip install it via a wheel package whereas `conda` does not use wheel, instead it creates its own package, currently the playwright cli is bundled into the wheel hence it works but it won't work in `conda`

**Comment by sugizo** [comment]
> Yes, It works with pip because pip install it via a wheel package whereas `conda` does not use wheel, instead it creates its own package, currently the playwright cli is bundled into the wheel hence it works but it won't work in `conda`

not sure understand this
**steps**
```
$ pip install playwright
$ pip list | grep playwright
playwright                    0.162.2
$ playwright codegen
-bash: playwright: command not found
```
think its installed via 
`pip install `
not from 
`conda install `
command

**Comment by holg** [comment]
i am using playwright v1.8.0a1, you can install it with pip: 
`pip install playwright==1.8.0a1`

**Comment by sugizo** [comment]
```
$ pip install playwright==1.8.0a1
ERROR: Could not find a version that satisfies the requirement playwright==1.8.0a1
ERROR: No matching distribution found for playwright==1.8.0a1
```

**Comment by holg** [comment]
ok, that's weird, i don't know of limitations of the python.
Maybe this helps:
conda update pip               
Collecting package metadata (current_repodata.json): done
Solving environment: done

# All requested packages already installed.

```
(/Users/htr/Documents/develeop/selenium/condaselenium) ➜  bin pip install playwright==1.8.0a1
Requirement already satisfied: playwright==1.8.0a1 in /Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/site-packages (1.8.0a1)
Requirement already satisfied: pyee>=8.0.1 in /Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/site-packages (from playwright==1.8.0a1) (8.1.0)
Requirement already satisfied: greenlet==1.0.0 in /Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/site-packages (from playwright==1.8.0a1) (1.0.0)
Requirement already satisfied: typing-extensions in /Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/site-packages (from playwright==1.8.0a1) (3.7.4.3)
(/Users/htr/Documents/develeop/selenium/condaselenium) ➜  bin conda -V
conda 4.9.2
```

**Comment by holg** [comment]
you also can install it directly from GitHub:
`pip install git+https://github.com/microsoft/playwright-python.git@release-1.8`

`pip install git+https://github.com/microsoft/playwright-python.git` would install the lastest 1.9
this seems to make some problems on the mac:
```
pip install git+https://github.com/microsoft/playwright-python.git
Collecting git+https://github.com/microsoft/playwright-python.git
  Cloning https://github.com/microsoft/playwright-python.git to /private/var/folders/nl/0myxft0551737nbr9zsmqd840000gn/T/pip-req-build-1gi300co
Requirement already satisfied: greenlet==1.0.0 in /Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/site-packages (from playwright==1.9.0.post5+g246aed8) (1.0.0)
Requirement already satisfied: pyee>=8.0.1 in /Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/site-packages (from playwright==1.9.0.post5+g246aed8) (8.1.0)
Requirement already satisfied: typing-extensions in /Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/site-packages (from playwright==1.9.0.post5+g246aed8) (3.7.4.3)
Building wheels for collected packages: playwright
  Building wheel for playwright (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/htr/Documents/develeop/selenium/condaselenium/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/nl/0myxft0551737nbr9zsmqd840000gn/T/pip-req-build-1gi300co/setup.py'"'"'; __file__='"'"'/private/var/folders/nl/0myxft0551737nbr9zsmqd840000gn/T/pip-req-build-1gi300co/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/nl/0myxft0551737nbr9zsmqd840000gn/T/pip-wheel-0tyrwr25
       cwd: /private/var/folders/nl/0myxft0551737nbr9zsmqd840000gn/T/pip-req-build-1gi300co/
  Complete output (197 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib
  creating build/lib/playwright
  copying playwright/_repo_version.py -> build/lib/playwright
  copying playwright/__init__.py -> build/lib/playwright
  copying playwright/__main__.py -> build/lib/playwright
  running egg_info
  creating playwright.egg-info
  writing playwright.egg-info/PKG-INFO
  writing dependency_links to playwright.egg-info/dependency_links.txt
  writing entry points to playwright.egg-info/entry_points.txt
  writing requirements to playwright.egg-info/requires.txt
  writing top-level names to playwright.egg-info/top_level.txt
  writing manifest file 'playwright.egg-info/SOURCES.txt'
  writing manifest file 'playwright.egg-info/SOURCES.txt'
  copying playwright/py.typed -> build/lib/playwright
  creating build/lib/playwright/_impl
  copying playwright/_impl/_accessibility.py -> build/lib/playwright/_impl
  copying playwright/_impl/_api_structures.py -> build/lib/playwright/_impl
  copying playwright/_impl/_api_types.py -> build/lib/playwright/_impl
  copying playwright/_impl/_async_base.py -> build/lib/playwright/_impl
  copying playwright/_impl/_browser.py -> build/lib/playwright/_impl
  copying playwright/_impl/_browser_context.py -> build/lib/playwright/_impl
  copying playwright/_impl/_browser_type.py -> build/lib/playwright/_impl
  copying playwright/_impl/_cdp_session.py -> build/lib/playwright/_impl
  copying playwright/_impl/_chromium_browser_context.py -> build/lib/playwright/_impl
  copying playwright/_impl/_connection.py -> build/lib/playwright/_impl
  copying playwright/_impl/_console_message.py -> build/lib/playwright/_impl
  copying playwright/_impl/_dialog.py -> build/lib/playwright/_impl
  copying playwright/_impl/_download.py -> build/lib/playwright/_impl
  copying playwright/_impl/_driver.py -> build/lib/playwright/_impl
  copying playwright/_impl/_element_handle.py -> build/lib/playwright/_impl
  copying playwright/_impl/_event_context_manager.py -> build/lib/playwright/_impl
  copying playwright/_impl/_file_chooser.py -> build/lib/playwright/_impl
  copying playwright/_impl/_frame.py -> build/lib/playwright/_impl
  copying playwright/_impl/_helper.py -> build/lib/playwright/_impl
  copying playwright/_impl/_impl_to_api_mapping.py -> build/lib/playwright/_impl
  copying playwright/_impl/_input.py -> build/lib/playwright/_impl
  copying playwright/_impl/_js_handle.py -> build/lib/playwright/_impl
  copying playwright/_impl/_network.py -> build/lib/playwright/_impl
  copying playwright/_impl/_object_factory.py -> build/lib/playwright/_impl
  copying playwright/_impl/_page.py -> build/lib/playwright/_impl
  copying playwright/_impl/_path_utils.py -> build/lib/playwright/_impl
  copying playwright/_impl/_playwright.py -> build/lib/playwright/_impl
  copying playwright/_impl/_selectors.py -> build/lib/playwright/_impl
  copying playwright/_impl/_sync_base.py -> build/lib/playwright/_impl
  copying playwright/_impl/_transport.py -> build/lib/playwright/_impl
  copying playwright/_impl/_video.py -> build/lib/playwright/_impl
  copying playwright/_impl/_wait_helper.py -> build/lib/playwright/_impl
  creating build/lib/playwright/async_api
  copying playwright/async_api/__init__.py -> build/lib/playwright/async_api
  copying playwright/async_api/_context_manager.py -> build/lib/playwright/async_api
  copying playwright/async_api/_generated.py -> build/lib/playwright/async_api
  creating build/lib/playwright/sync_api
  copying playwright/sync_api/__init__.py -> build/lib/playwright/sync_api
  copying playwright/sync_api/_context_manager.py -> build/lib/playwright/sync_api
  copying playwright/sync_api/_generated.py -> build/lib/playwright/sync_api
  installing to build/bdist.macosx-10.9-x86_64/wheel
  running install
  running install_lib
  creating build/bdist.macosx-10.9-x86_64
  creating build/bdist.macosx-10.9-x86_64/wheel
  creating build/bdist.macosx-10.9-x86_64/wheel/playwright
  creating build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_accessibility.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_driver.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_cdp_session.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_selectors.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_playwright.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_helper.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_api_types.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_download.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_console_message.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_transport.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_impl_to_api_mapping.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_browser_context.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_page.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_event_context_manager.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_video.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_path_utils.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_browser.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_network.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_sync_base.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_browser_type.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_api_structures.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_chromium_browser_context.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_dialog.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_input.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_frame.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_file_chooser.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_js_handle.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_async_base.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_element_handle.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_object_factory.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_wait_helper.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_impl/_connection.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/_impl
  copying build/lib/playwright/_repo_version.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright
  creating build/bdist.macosx-10.9-x86_64/wheel/playwright/async_api
  copying build/lib/playwright/async_api/_context_manager.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/async_api
  copying build/lib/playwright/async_api/_generated.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/async_api
  copying build/lib/playwright/async_api/__init__.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/async_api
  copying build/lib/playwright/__init__.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright
  creating build/bdist.macosx-10.9-x86_64/wheel/playwright/sync_api
  copying build/lib/playwright/sync_api/_context_manager.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/sync_api
  copying build/lib/playwright/sync_api/_generated.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/sync_api
  copying build/lib/playwright/sync_api/__init__.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright/sync_api
  copying build/lib/playwright/py.typed -> build/bdist.macosx-10.9-x86_64/wheel/playwright
  copying build/lib/playwright/__main__.py -> build/bdist.macosx-10.9-x86_64/wheel/playwright
  running install_egg_info
  Copying playwright.egg-info to build/bdist.macosx-10.9-x86_64/wheel/playwright-1.9.0.post5+g246aed8-py3.8.egg-info
  running install_scripts
  adding license file "LICENSE" (matched pattern "LICEN[CS]E*")
  creating build/bdist.macosx-10.9-x86_64/wheel/playwright-1.9.0.post5+g246aed8.dist-info/WHEEL
  creating '/private/var/folders/nl/0myxft0551737nbr9zsmqd840000gn/T/pip-wheel-0tyrwr25/playwright-1.9.0.post5+g246aed8-py3-none-any.whl' and adding 'build/bdist.macosx-10.9-x86_64/wheel' to it
  adding 'playwright/__init__.py'
  adding 'playwright/__main__.py'
  adding 'playwright/_repo_version.py'
  adding 'playwright/py.typed'
  adding 'playwright/_impl/_accessibility.py'
  adding 'playwright/_impl/_api_structures.py'
  adding 'playwright/_impl/_api_types.py'
  adding 'playwright/_impl/_async_base.py'
  adding 'playwright/_impl/_browser.py'
  adding 'playwright/_impl/_browser_context.py'
  adding 'playwright/_impl/_browser_type.py'
  adding 'playwright/_impl/_cdp_session.py'
  adding 'playwright/_impl/_chromium_browser_context.py'
  adding 'playwright/_impl/_connection.py'
  adding 'playwright/_impl/_console_message.py'
  adding 'playwright/_impl/_dialog.py'
  adding 'playwright/_impl/_download.py'
  adding 'playwright/_impl/_driver.py'
  adding 'playwright/_impl/_element_handle.py'
  adding 'playwright/_impl/_event_context_manager.py'
  adding 'playwright/_impl/_file_chooser.py'
  adding 'playwright/_impl/_frame.py'
  adding 'playwright/_impl/_helper.py'
  adding 'playwright/_impl/_impl_to_api_mapping.py'
  adding 'playwright/_impl/_input.py'
  adding 'playwright/_impl/_js_handle.py'
  adding 'playwright/_impl/_network.py'
  adding 'playwright/_impl/_object_factory.py'
  adding 'playwright/_impl/_page.py'
  adding 'playwright/_impl/_path_utils.py'
  adding 'playwright/_impl/_playwright.py'
  adding 'playwright/_impl/_selectors.py'
  adding 'playwright/_impl/_sync_base.py'
  adding 'playwright/_impl/_transport.py'
  adding 'playwright/_impl/_video.py'
  adding 'playwright/_impl/_wait_helper.py'
  adding 'playwright/async_api/__init__.py'
  adding 'playwright/async_api/_context_manager.py'
  adding 'playwright/async_api/_generated.py'
  adding 'playwright/sync_api/__init__.py'
  adding 'playwright/sync_api/_context_manager.py'
  adding 'playwright/sync_api/_generated.py'
  adding 'playwright-1.9.0.post5+g246aed8.dist-info/LICENSE'
  adding 'playwright-1.9.0.post5+g246aed8.dist-info/METADATA'
  adding 'playwright-1.9.0.post5+g246aed8.dist-info/WHEEL'
  adding 'playwright-1.9.0.post5+g246aed8.dist-info/entry_points.txt'
  adding 'playwright-1.9.0.post5+g246aed8.dist-info/top_level.txt'
  adding 'playwright-1.9.0.post5+g246aed8.dist-info/RECORD'
  removing build/bdist.macosx-10.9-x86_64/wheel
  Fetching  https://playwright.azureedge.net/builds/driver/next/playwright-1.9.0-1614037901000-mac.zip
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                   Dload  Upload   Total   Spent    Left  Speed
  100 16.2M  100 16.2M    0     0   626k      0  0:00:26  0:00:26 --:--:--  614k
  Fetching  https://playwright.azureedge.net/builds/driver/next/playwright-1.9.0-1614037901000-linux.zip
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                   Dload  Upload   Total   Spent    Left  Speed
  100 18.3M  100 18.3M    0     0   615k      0  0:00:30  0:00:30 --:--:--  622k
  Fetching  https://playwright.azureedge.net/builds/driver/next/playwright-1.9.0-1614037901000-win32.zip
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                   Dload  Upload   Total   Spent    Left  Speed
  100 11.7M  100 11.7M    0     0   693k      0  0:00:17  0:00:17 --:--:--  715k
  Fetching  https://playwright.azureedge.net/builds/driver/next/playwright-1.9.0-1614037901000-win32_x64.zip
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                   Dload  Upload   Total   Spent    Left  Speed
  100 13.3M  100 13.3M    0     0   605k      0  0:00:22  0:00:22 --:--:--  609k
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/private/var/folders/nl/0myxft0551737nbr9zsmqd840000gn/T/pip-req-build-1gi300co/setup.py", line 100, in <module>
      setuptools.setup(
    File "/Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/site-packages/setuptools/__init__.py", line 153, in setup
      return distutils.core.setup(**attrs)
    File "/Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/distutils/core.py", line 148, in setup
      dist.run_commands()
    File "/Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/distutils/dist.py", line 966, in run_commands
      self.run_command(cmd)
    File "/Users/htr/Documents/develeop/selenium/condaselenium/lib/python3.8/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/private/var/folders/nl/0myxft0551737nbr9zsmqd840000gn/T/pip-req-build-1gi300co/setup.py", line 58, in run
      base_wheel_location = glob.glob("dist/*.whl")[0]
  IndexError: list index out of range
  ----------------------------------------
  ERROR: Failed building wheel for playwright
  Running setup.py clean for playwright
Failed to build playwright
```

**Comment by sugizo** [comment]
```
$ pip install git+https://github.com/microsoft/playwright-python.git@release-1.8
$ playwright codegen
Traceback (most recent call last):
  File "/Users/sugizo/miniconda3/envs/python3_test/bin/playwright", line 33, in <module>
    sys.exit(load_entry_point('playwright==1.8.0a1', 'console_scripts', 'playwright')())
  File "/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/site-packages/playwright/__main__.py", line 28, in main
    subprocess.run([str(driver_executable), *sys.argv[1:]], env=my_env)
  File "/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/subprocess.py", line 489, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/site-packages/playwright/driver/playwright.sh'
```

**Comment by holg** [comment]
did you try:

`playwright install`

this shall install the binaries needed, e.g. Chromium and driver stuff

**Comment by sugizo** [comment]
```
$ playwright install
Traceback (most recent call last):
  File "/Users/sugizo/miniconda3/envs/python3_test/bin/playwright", line 33, in <module>
    sys.exit(load_entry_point('playwright==1.8.0a1', 'console_scripts', 'playwright')())
  File "/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/site-packages/playwright/__main__.py", line 28, in main
    subprocess.run([str(driver_executable), *sys.argv[1:]], env=my_env)
  File "/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/subprocess.py", line 489, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugizo/miniconda3/envs/python3_test/lib/python3.8/site-packages/playwright/driver/playwright.sh'
```
even 
`playwright --help`
produce same error result

**Comment by mxschmitt** [comment]
Closed as a duplicate of #517
